confidant/routes/v1.py

```diff
@@ -45,9 +45,9 @@ def get_user_info():
     Get the email address of the currently logged-in user.
     '''
     try:
-        response = jsonify({'email': authnz.get_logged_in_user()})
+        response = jsonify({'email': authnz.get_logged_in_user(), 'groups': authnz.get_user_groups()})
     except authnz.UserUnknownError:
-        response = jsonify({'email': None})
+        response = jsonify({'email': None, 'groups': None})
```

Comment:
I'm confused about this diff, because `confidant/routes/v1.py` doesn't exist anymore. It's been split up into files specific to the types of endpoints.